src/cascadia/TerminalApp/ColorPickupFlyout.cpp

```diff
@@ -91,4 +76,20 @@ namespace winrt::TerminalApp::implementation
     {
         _ColorSelectedHandlers(args.NewColor());
     }
+
+    void ColorPickupFlyout::Pivot_SelectionChanged(const IInspectable&, const Windows::UI::Xaml::Controls::SelectionChangedEventArgs& args)
+    {
+        // Pivot likes to take up as much width as possible (as opposed to just what it needs from its children).
+        // Using trial-and-error, we've determined a reasonable width for each pivot item.
+        const auto selectedItem = args.AddedItems().GetAt(0).as<Windows::UI::Xaml::Controls::PivotItem>();
+        const auto tag = unbox_value<hstring>(selectedItem.Tag());
+        if (tag == L"Standard")
+        {
+            FlyoutPivot().Width(170);
+        }
+        else if (tag == L"Custom")
+        {
+            FlyoutPivot().Width(340);
```

Comment:
Hmmm...... Hardcoded widths. 🤔 I'm not so sure that will work fine with i18n and such. 😅

Can't the XAML control size itself as needed?

Reply:
Urgh. You're 100% right.

I tried doing this the right way by...
1. set the `Pivot`'s width to `auto` --> the pivot ends up taking up the max width instead of the necessary width for it's children
2. in this function, set the width on a `LayoutUpdated` or `SizeChanged` event to the `ActualWidth` or `DesiredWidth` of the children --> a combination of issues where the width wouldn't be set properly or it wouldn't be ready in time

I'll have to look more into how the layout/sizing process works in XAML. Ugh.